### PR TITLE
WIP: Remove the default gRPC configuration for the nginx-agent

### PR DESCRIPTION
**Pull request by oliveromahony**
### Proposed changes
Remove the default gRPC configuration. This uses REST by default

### Checklist

Before creating a PR, run through this checklist and mark each as complete.

- [x] I have read the [`CONTRIBUTING`](https://github.com/nginx/agent/blob/main/docs/CONTRIBUTING.md) document
- [x] If applicable, I have added tests that prove my fix is effective or that my feature works
- [x] If applicable, I have checked that any relevant tests pass after adding my changes
- [x] I have updated any relevant documentation ([`README.md`](https://github.com/nginx/agent/blob/main/README.md))

### CODE REVIEW

**Review by Jcahilltorre — APPROVED** [review]

**Review by Dean-Coakley — APPROVED** [review]

**Review by dhurley — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -155,7 +155,7 @@ sudo touch /etc/nginx-agent/agent-dynamic.conf
 ```
 
 ### Enabling the gRPC interface
-Update the `/etc/nginx-agent/nginx-agent.conf` file to include the following settings:
+Add the the following settings to `/etc/nginx-agent/nginx-agent.conf`:
```
Comment (mtbChef) [inline README.md:158]: I think there's a typo. The line seems to read "Add the the..." now

File: docs/content/getting-started.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ sudo touch /etc/nginx-agent/agent-dynamic.conf
 
 ### Enabling the gRPC interface
 
-Update the `/etc/nginx-agent/nginx-agent.conf` file to include the following settings:
+Add the the following settings to `/etc/nginx-agent/nginx-agent.conf`:
```
Comment (mtbChef) [inline docs/content/getting-started.md:51]: Same thing here

**Review by mtbChef — CHANGES_REQUESTED** [review]
Minor typo in the readme

File: README.md (hunk 1 above)
Comment (Jcahilltorre) [reply README.md:158]:
```suggestion
Add the following settings to `/etc/nginx-agent/nginx-agent.conf`:
```

File: docs/content/getting-started.md (hunk 2 above)
Comment (Jcahilltorre) [reply docs/content/getting-started.md:51]:
```suggestion
Add the following settings to `/etc/nginx-agent/nginx-agent.conf`:
```

**Review by mtbChef — APPROVED** [review]

**Pull request merged**